payment/src/main/java/org/killbill/billing/payment/core/ProcessorBase.java

```diff
@@ -204,16 +210,27 @@ public WithAccountLock(final PaymentConfig paymentConfig) {
         }
     }
 
-    protected static <ReturnType> ReturnType dispatchWithExceptionHandling(@Nullable final Account account, final Callable<PluginDispatcherReturnType<ReturnType>> callable, PluginDispatcher<ReturnType> pluginFormDispatcher) throws PaymentApiException {
+    protected static <ReturnType> ReturnType dispatchWithExceptionHandling(@Nullable final Account account, final String pluginName, final Callable<PluginDispatcherReturnType<ReturnType>> callable, PluginDispatcher<ReturnType> pluginFormDispatcher) throws PaymentApiException {
         final UUID accountId = account != null ? account.getId() : null;
         final String accountExternalKey = account != null ? account.getExternalKey() : "";
+
+        final String requestId = Request.getPerThreadRequestData() != null
+                                 ? Request.getPerThreadRequestData().getRequestId() : "NotAvailableRequestId";
+
         try {
-            return pluginFormDispatcher.dispatchWithTimeout(callable);
+            log.info("Calling plugin {} with requestId {}", pluginName, requestId);
```

Comment:
This seems a bit too much logging. After talking to @sbrossie, can you put `debug` instead?